app/containers/Sessions/components/QuestionTypeChooser/index.tsx

```diff
@@ -172,15 +182,16 @@ const QuestionTypeChooser = ({
                   conditionalAppearanceConfig={conditionalAppearanceConfigs[id]}
                 />
               ))}
-              {AddableGroups.map(({ color, id }) => (
-                <NewItem
-                  key={id}
-                  color={color}
-                  handleClick={() => handleClick(id)}
-                  title={formatMessage(questionGroupTypeMessages[id])}
-                  isGroup
-                />
-              ))}
+              {sessionType !== SessionTypes.SMS_SESSION &&
```

Comment:
What about cat_mh session type? Shouldn't it be `sessionType === SessionTypes.CLASSIC_SESSION || sessionType === CAT_MH`?

Reply:
Because in this type of session researchers can't define the content of session by adding questions - the questions are fetched from CAT-MH database